examples/forecasts/basic_example_forecasting.py

```diff
@@ -0,0 +1,56 @@
+from forecasting.models.gluonts_model.mqcnn.mqcnn_model import MQCNNModel
+from forecasting.task.forecasting.dataset import TimeSeriesDataset
+
+import core as ag
+from core.scheduler import FIFOScheduler
+
+dataset = TimeSeriesDataset(
+    train_path="./COV19/processed_train.csv",
+    test_path="./COV19/processed_test.csv",
+    index_column="name",
+    target_column="ConfirmedCases",
+    time_column="Date")
+
+
+@ag.args(context_length=ag.Int(1, 20))
+def train_fn(args, reporter):
+    context_length = args.context_length
+    train_model = MQCNNModel(path="model/", hyperparameters={"context_length": context_length})
+    train_model.fit(dataset.train_data)
+    for e in range(3):
+        train_score = train_model.score(dataset.test_data)
```

Comment:
this is not training score, should be instead called something like:
`validation_score`

Reply:
changed to val_score